sdk/core/core/src/main.c

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Should we move this file to `test` folder?

Reply:
yeah, what I want is to see the Pipeline working for Windows check.. But I don't see checks coming up

Reply:
Update: move the curl easy perform to test.
We can remove it later. For now it is a simple example on how to use curl.  CI is working for all platforms now.  On windows, it works on Visual Studio